### PR TITLE
update prometheus-config to new format

### PR DESCRIPTION
**Pull request by krombel**
Prometheus changed its format from simple config file to YAML for version 2.0.0 ([changelog](https://github.com/prometheus/prometheus/blob/master/CHANGELOG.md#200--2017-11-08))
But the configs provided by this repo does not reflect that change.

Signed-Off-By: Matthias Kesler <krombel@krombel.de>

### CODE REVIEW

File: contrib/prometheus/README [hunk 1]
```diff
@@ -5,15 +5,20 @@ To use it, first install prometheus by following the instructions at
 
   http://prometheus.io/
 
-Then add a new job to the main prometheus.conf file:
+Then add a new job to the main prometheus.yml file:
```
Comment (richvdh) [inline contrib/prometheus/README]: prometheus.yaml, no?

Comment (krombel) [reply contrib/prometheus/README]: `prometheus.yml` is working in my running setup

Comment (richvdh) [reply contrib/prometheus/README]: `prometheus.yaml` is working in my running setup

**Comment by richvdh** [comment]
how about separate "contrib/prometheus-v1" and "contrib/prometheus-v2" directories? I think that woudl be clearer

**Comment by krombel** [comment]
The differences are only in different files for "synapse.rules" and some adjustment in the README. I can create two directories with two files each (README+synapse.rules) but I do not see the benefit...

But if you prefer it I will do it

**Comment by richvdh** [comment]
hrm, if it wasn't for the console, then there would hardly be anything shared between v1 and v2

but fair enough

**Comment by krombel** [comment]
Just for refence: I had to checkout the templates of the repos to use synapse.html that is in this repo. But synapse.html was usable without modification since I did that.

**Comment by richvdh** [comment]
the templates of the repos?

**Comment by krombel** [comment]
This directory: https://github.com/prometheus/prometheus/tree/7ebd74b/console_libraries

**Pull request merged**